src/components/picker/PickerItem.js

```diff
@@ -80,10 +80,6 @@ const PickerItem = props => {
     );
   };
 
-  if (context.showSearch && shouldFilterOut(context.searchValue, itemLabel)) {
```

Comment:
You can also stop passing `showSearch` and `searchValue` in the context provided by the Picker

Reply:
Done